### PR TITLE
[ru] Localize cloud-native-security.md

### PR DESCRIPTION
**Pull request by kirkonru**
### Description

This PR adds RU localization of **Cloud Native Security**

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/ru/cloud-native-security.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+tags: ["security", "", ""]
+---
+
+## Описание
+
+Нативная облачная безопасность — подход, при котором безопасность встраивается в [нативные облачные приложения](/ru/cloud-native-apps/).
+В его рамках безопасность становится частью всего жизненного цикла приложения: от разработки до production.
+Нативная облачная безопасность сред направлена на обеспечение тех же стандартов, что и традиционные модели безопасности, 
+но при этом адаптирована к особенностям облачных окружений, 
+а именно, к быстрым изменениям кода и эфемерной инфраструктуре.
```
Comment (shurup) [inline content/ru/cloud-native-security.md:14]: This seems to be the "Перед однородными членами предложения" case explained [here](https://pishugramotno.ru/punktuacia/a-imenno-kogda-stavim-zapyatuyu-dvoetochie-ili-tire).
```suggestion
а именно: быстрым изменениям кода и эфемерной инфраструктуре.
```

**Review by shurup — APPROVED** [review]
I have reviewed this term in #2470 before. Confirming the text is intact and inviting @tym83 to review it, too.

File: content/ru/cloud-native-security.md (hunk 1 above)
Comment (kirkonru) [reply content/ru/cloud-native-security.md:14]: Yep, thanks!

**Comment by kirkonru** [comment]
@tym83 please approve 🥺

**Review by tym83 — APPROVED** [review]

**Pull request merged**